### PR TITLE
feat(space): add play_game external

### PR DESCRIPTION
**Pull request by tdelabro**

### CODE REVIEW

**Review by abdelhamidbakhta — APPROVED** [review]
lgtm

File: contracts/core/space.cairo [hunk 1]
```diff
@@ -166,6 +170,27 @@ end
...
+func play_game{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr, bitwise_ptr : BitwiseBuiltin*}():
+
+    _rec_play_turns()
+
+    return ()
+end
+
+func _rec_play_turns{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr, bitwise_ptr : BitwiseBuiltin*}():
+    let (is_finished) = next_turn()
+    if is_finished != 0:
```
Comment (ofux) [inline contracts/core/space.cairo:183]:
```suggestion
    if is_finished == TRUE:
```

File: tests/test_space.cairo [hunk 2]
```diff
@@ -7,8 +7,48 @@ from starkware.cairo.common.bool import TRUE, FALSE
 from starkware.starknet.common.syscalls import get_contract_address, get_caller_address
 from protostar.asserts import assert_eq
 
+# @external
```
Comment (ofux) [inline tests/test_space.cairo:10]: why commented?

**Review by Bernardstanislas — APPROVED** [review]

**Review by ofux — APPROVED** [review]

File: tests/test_space.cairo (hunk 2 above)
Comment (tdelabro) [reply tests/test_space.cairo:10]: It doesn't pass since a few push. Even on master

File: contracts/core/space.cairo (hunk 1 above)
Comment (tdelabro) [reply contracts/core/space.cairo]: Done

File: tests/test_space.cairo (hunk 2 above)
Comment (tdelabro) [reply tests/test_space.cairo:10]: Will fix later. In other PR

**Pull request merged**